skills/csharp/experimental/itsmskill/Utilities/AdaptiveCardHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace ITSMSkill.Utilities
+{
+    using System.IO;
+    using AdaptiveCards;
+    using Newtonsoft.Json;
+
+    public class AdaptiveCardHelper
+    {
+        public static AdaptiveCard GetCardFromJson(string jsonFile)
```

Comment:
I am still encouraging to use lg for response (text, card etc.) as 1 it is the common approach now 2 it supports multi-language better

Reply:
Added a TODO to use Cards.Lg and responses. I will create a seperate PR for that purpose.